assets/js/pages/article/read/ReadArticle.vue

```diff
@@ -4,10 +4,12 @@
              v-if="$store.getters.getCurrentArticle"
              :key="$store.getters.getCurrentArticle.file.path"
              v-hotkey="keymap">
-
             <a-row type="flex">
                 <a-col class="article-content-col"
                        :style="{ width : $store.getters.getTocCollapsed ? '100%' : 'calc(100% - 250px)' }">
+                    <article-title v-if="$store.getters.getCurrentArticle.file.metadata.title">
+                        {{ $store.getters.getCurrentArticle.file.metadata.title }}
+                    </article-title>
```

Comment:
It seems that we use the ArticleTitle component only here..., why creating this component if it's just one `h1` html tag ? If we use it somewhere else, it's alright... But still, why passing a props then ? Couldn't we just display the value inside the component if the value displayed will always be equal to `this.$store.getters.getCurrentArticle.file.metadata.title` ?